home-work-2/style.css

```diff
@@ -0,0 +1,194 @@
+html {
+    height: 100%;
+}
+
+.body {
+    background-color: #f9f9f9;
+}
+
+.container {
+    -moz-box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.3);
+    -webkit-box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.3);
+    background-color: #fff;
+    box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.3);
+    box-sizing: border-box;
+    height: 100%;
+    margin: auto;
+    padding: 40px;
+    width: 1060px;
+    position: relative;
+}
+
+/*HEADER*/
+
+#logo {
+    padding-bottom: 40px;
+}
+
+.main-menu {
+    background: linear-gradient(0deg, rgba(68,67,67,1) 0%, rgba(94,94,94,1) 35%);
+    font-family: 'Open Sans';
+    font-size: 15px;
+    height: 40px;
+    width: 100%;
+}
+
+.main-menu-list {
+    display: table;
+    height: 100%;
+}
+
+.main-menu-list li {
+    border-left: 1px solid transparent;
+    border-right: 1px solid transparent;
+    display: table-cell;
+    height: 100%;
+    padding: 0 20px;
+    vertical-align: middle;
+}
+
+.main-menu-list li:hover {
+    background: linear-gradient(0deg, rgba(46,46,46,1) 0%, rgba(74,74,74,1) 35%);
+    background: rgb(46,46,46);
+    border-left: 1px solid #838383;
+    border-right: 1px solid #838383;
+    cursor: pointer;
+    padding: 0 20px;
+}
+
+.main-menu-list a {
+    color: #fff;
+    text-decoration: none;
+}
+
+/*the end of HEADER*/
+
+/*MAIN-BLOCK*/
+
+.main-block {
+    overflow: auto;
+    padding: 40px 0px;
+    width: 100%;
+    margin-bottom: 40px;
+}
+
+
+/*SIDEBAR*/
+
+.sidebar {
+    border-left: 1px solid #c8c8c8;
+    border-right: 1px solid #c8c8c8;
+    float: left;
+    width: 240px;
+}
+
+.sidebar-list li {
+    border-bottom: 1px solid #c8c8c8;
+    font-family: "Open Sans";
+    font-size: 14px;
+    line-height: 40px;
+}
+
+.sidebar-list li a {
+    color: #656363;
+    display: block;
+    height: 100%;
+    padding-left: 20px;
+    text-decoration: none;
+    width: calc(100% - 20px);
+}
+
+.sidebar-list li:first-child {
+    border-top: 1px solid #c8c8c8;
+}
+
+.sidebar-list li:hover {
+    background-color: #f9f9f9;
+    box-sizing: border-box;
+    color: #2c2c2c;
+    cursor: pointer;
+    padding-left: 20px;
+}
+
+/*end of SIDEBAR*/
+
+/*PRODUCT*/
+.product-wraper {
+    float: right;
+    width: 678px;
+}
+
+.product-img {
+    border: 1px solid #c8c8c8;
+    display: flex;
+    justify-content: center;
+    overflow: hidden;
+}
+
+.product-description {
+    color: #2c2c2c;
+    font-family: "Open Sans";
+    font-size: 14px;
+    line-height: 18px;
+    padding-left: 24px;
+    padding-top: 40px;
+}
+
+/*the end of PRODUCT*/
+
+/*FOOTER*/
+.footer {
+    width: 100%;
+    background-color: #e1e1e1;
+    height: 40px;
+    clear: both;
+    position: absolute;
```

Comment:
please remove content of product-description block and check if footer is still in the bottom of page